app/javascript/app/acuant/document_capture_dom.js

```diff
@@ -28,6 +29,16 @@ const hideAcuantSdkContainers = () => {
   acuantSdkContinueForm().classList.add('hidden');
 };
 
+export const hideAcuantSdkCaptureView = () => {
+  acuantSdkCaptureView().classList.add('hidden');
+  acuantSdkUploadForm().classList.remove('hidden');
```

Comment:
I think you could get the same result as the two lines by calling `showAcuantSdkContainer('upload-form');`